login/template.ftl

```diff
@@ -29,6 +29,19 @@
             <script src="${script}" type="text/javascript"></script>
         </#list>
     </#if>
+    <script type="text/javascript">
+        var _paq = window._paq || [];
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        _paq.push(["setUserId", "REPLACE THIS WITH THE USER ID, IF KNOWN"]);
```

Comment:
Unless I'm mistake, I think the login page is the only keycloak page a staff user would access during a normal login. Unfortunately, we won't know the user's ID before they log in, so I don't think setting this variable will be too useful.

Please let me know if I've overlooked any situation/mechanism where we'd be able to collect this!

(I may be missing other ways of grabbing the current user, but I also couldn't find any pre-defined javascript variables set by keycloak)
```suggestion
```